src/commands/clean.ts

```diff
@@ -0,0 +1,16 @@
+import * as fs from "fs-extra";
+import * as lib from "../lib";
+import * as oclif from "@oclif/command";
+
+export default class Clean extends oclif.Command {
+    static description = "clean all JS files without matching TS files";
+
+    async run() {
+        const files = (await lib.recursiveReaddir(lib.WORKING_DIR))
+            .map(f => f.replace(/\\/g, "/"))
+            .filter(f => !f.includes("/node_modules/"));
+        const unmatchedFiles = files.filter(f => f.endsWith(".js") && !files.includes(f.replace(/\.js/g, ".ts")));
```

Comment:
I don't think it would alter functionality (I've never accidentally left one by itself, so idk what it would do) but could this include removal of `.js.map` files as well? Could potentially just check if the file includes instead of ends with `.js` to cover both cases.

Reply:
Map files are pretty innocuous - they don't ever cause harm by existing, so I'm not worried about it. It'd be a minor pain to include support for them, which is annoying.